doc/classes/Array.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Array" version="4.0">
 	<brief_description>
-		Generic array datatype.
+		A Generic array datatype.
```

Comment:
This adds a typo :)
"generic" should be made lowercase (same in long description).